src/Examine.Lucene/Search/LuceneSearchResults.cs

```diff
@@ -1,23 +1,27 @@
-﻿using System;
+using System;
 using System.Collections;
 using System.Collections.Generic;
+using System.Collections.ObjectModel;
 
 namespace Examine.Lucene.Search
 {
-    public class LuceneSearchResults : ISearchResults
+    public class LuceneSearchResults : ISearchResults, IFacetResults
     {
-        public static LuceneSearchResults Empty { get; } = new LuceneSearchResults(Array.Empty<ISearchResult>(), 0);
+        public static LuceneSearchResults Empty { get; } = new LuceneSearchResults(Array.Empty<ISearchResult>(), 0, new ReadOnlyDictionary<string, IFacetResult>(new Dictionary<string, IFacetResult>()));
```

Comment:
You shouldn't need to create a ReadOnlyDictionary wrapping a new dictionary since Dictionary<string, IFacetResult> already implements IReadOnlyDictionary<string, IFacetResult>